docs/src/inscriptions/recursion.md

```diff
@@ -26,11 +26,70 @@ The recursive endpoints are:
 - `/r/blockhash`: latest block hash.
 - `/r/blockhash/<HEIGHT>`: block hash at given block height.
 - `/r/blocktime`: UNIX time stamp of latest block.
-- `/r/metadata/<INSCRIPTION_ID>`: returns a JSON string containing the hex-encoded CBOR metadata.
+- `/r/metadata/<INSCRIPTION_ID>`: JSON string containing the
+  hex-encoded CBOR metadata.
+- `/r/sat/<SAT_NUMBER>`: the first 100 inscription ids on a sat.
+- `/r/sat/<SAT_NUMBER>/<PAGE>`: the set of 100 inscription ids on `<PAGE>`.
+- `/r/sat/<SAT_NUMBER>/at/<INDEX>`: the inscription id at `<INDEX>` of all
+  inscriptions on a sat. `<INDEX>` may be a negative number to index from the
+  back.
```

Comment:
Should probaly be more explicit about the offby one on the negative number. 
0 first inscription -1 most recent